canvas/fizz-27-py/src/lib/assignments.py

```diff
@@ -0,0 +1,10 @@
+from lib.canvasRequest import get
+
+def getAssignments(courseId, assignmentStatus):
+    assignments = get(f"/api/v1/courses/{courseId}/assignments?bucket={assignmentStatus}")
+    return assignments
+
```

Comment:
You should set up Black to auto format your code:  https://github.com/psf/black.  For example, Python coding conventions apparently want two lines between function declarations.